js/src/components/paid-ads/asset-group/asset-group.js

```diff
@@ -79,6 +79,8 @@ export default function AssetGroup( { campaign } ) {
 			audiences: audiences.join( ',' ),
 			budget: values.amount.toString(),
 			assets_validation: isValidAssetGroup ? 'valid' : 'invalid',
+			campaign_id: isCreation ? 'new' : campaign?.id,
+			number_of_youtube_videos: values.youtube_video.length,
```

Comment:
@ankitrox This is being handled on [this line](https://github.com/woocommerce/google-listings-and-ads/blob/89f2f9f47e51cb07f5e2cc6d7fac01551b297544/js/src/components/paid-ads/asset-group/asset-group.js#L93).